core/src/acc_text/nlg/gf/service.clj

```diff
@@ -21,6 +21,14 @@
       (log/warnf "Unknown language `%s`, switching to `%s`" lang default-language)
       default-language)))
 
+(defn ping []
+  (let [service-host (or (System/getenv "GF_ENDPOINT") "http://localhost:8001")
+        request-url (str service-host "/health")
+        {request-error :error} @(client/request {:url request-url :method :get})]
+    (when (some? request-error)
+      (log/errorf "GF service host: %s is unreachable. Are you sure it is started and reachable?" service-host))
```

Comment:
The error printed in console is more descriptive now, but the indication in UI that something is wrong is gone.

Reply:
Not the best looking indication, but this is how it looked before (I actually never encountered this):
![Screenshot from 2021-02-19 11-05-33](https://user-images.githubusercontent.com/20424394/108482733-89607300-72a2-11eb-9fe6-a37662ae0883.png)